script/main.py

```diff
@@ -19,10 +19,18 @@ def create_driver(name):
   return driver
 
 
+def list_of_all_links(driver):
```

Comment:
This function should only return list of links.
Please provide separate function for printing or for now just print in function main.